src/ontology/slurp/doid.tsv

```diff
@@ -0,0 +1 @@
+	mondo_id	xref	label	definition	qualified_parents
```

Comment:
@matentzn All the output files being generated are empty. I imagine this is not expected. Perhaps we should look over the algorithm and inputs during our meeting, or live debug it.